README.rst

```diff
@@ -1377,7 +1377,7 @@ An example of ``output_fn`` for the accept type "application/python-pickle" can
 
     import numpy as np
 
-    def output_fn(data, accepts):
+    def output_fn(prediction_result, accepts):
         """An output_fn that dumps a pickled numpy as response"""
```

Comment:
change 'numpy' to 'object' for consistency with the above change?

Reply:
sure